jenkins/PerfCI_OpenShift_Deployment/Jenkinsfile

```diff
@@ -28,11 +28,29 @@ pipeline {
         stage('Cleanup') {
             steps {
                 script {
-                    sh '''
+                    try {
+                        // Clean the Jenkins workspace
+                        echo "Cleaning Jenkins workspace"
+                        deleteDir()
+
+                        // Delete benchmark-runner images
+                        sh '''
                         if [[ "$(sudo podman images -q quay.io/ebattat/benchmark-runner 2> /dev/null)" != "" ]]; then
                             sudo podman rmi -f $(sudo podman images -q quay.io/ebattat/benchmark-runner 2> /dev/null)
                         fi
-                    '''
+                        '''
+                        // Stop running containers
+                        sh 'sudo podman ps | awk \'{print $1}\' | sudo xargs -I % sudo podman stop % 2>/dev/null'
```

Comment:
All you should need is
```
sh 'sudo podman ps | awk \'{print $1}\' | sudo xargs podman stop 2>/dev/null'
```

xargs is running privileged, so anything that xargs spawns will similarly be running privileged.